website/docs/cloud-docs/overview/activate-flex.mdx

```diff
@@ -41,7 +41,7 @@ After you create your user accounts:
 
 Organization members may be owners or admins in only the Terraform Cloud or HCP organization. To safeguard access to your Flex balance, determine whether to invite Terraform Cloud organization owners to the HCP organization, or whether to invite HCP admins to Terraform Cloud.  
 
-**Example**: Multi-Tenant (i.e. Managed Services )
+**Example**: Multi-Tenant (i.e. Managed Services)
 
 MSP is an HCP admin and manages the billing relationship with HashiCorp Cloud Platform and services customers of Terraform Cloud.  The HCP admin should be invited to the Terraform Cloud customer organizations to activate Flex.
```

Comment:
Outside the scope of this PR, but what is MSP? Is it an acronym for a feature that we haven't defined on this page? I read it as a user name for the example scenario, hence my decision to suggest using a heading in the previous comment. 
```suggestion
MSP is an HCP admin and manages the billing relationship with HashiCorp Cloud Platform and services customers of Terraform Cloud.  The HCP admin should be invited to the Terraform Cloud customer organizations to activate Flex.
```